### PR TITLE
[racket/en] Expand list section

### PR DESCRIPTION
**Pull request by Derrreks**
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: racket.html.markdown [hunk 1]
```diff
@@ -8,6 +8,7 @@ contributors:
   - ["Gustavo Schmidt", "https://github.com/gustavoschmidt"]
   - ["Duong H. Nguyen", "https://github.com/cmpitg"]
   - ["Keyan Zhang", "https://github.com/keyanzhang"]
+  - ["Derek Schuster", "https://github.com/derrreks"]
```
Comment (samcv) [inline racket.html.markdown]: You should only add yourself to this list if you have contributed a larger section.

File: racket.html.markdown [hunk 2]
```diff
@@ -164,10 +165,16 @@ my-pet ; => #<dog>
 (cons 1 (cons 2 (cons 3 null))) ; => '(1 2 3)
 ;; `list' is a convenience variadic constructor for lists
 (list 1 2 3) ; => '(1 2 3)
-;; and a quote can also be used for a literal list value
+;; a quote can also be used for a literal list value
 '(1 2 3) ; => '(1 2 3)
+;; a quasiquote with commas can be used to evaluate functions
+`(1 ,(+ 1 1) 3) ; => '(1 2 3)
```
Comment (samcv) [inline racket.html.markdown]: Get rid of the ` mark here

Comment (Derrreks) [inline racket.html.markdown]: the ` is actually part of that line of code, similar to the ' from the line above

**Comment by Derrreks** [comment]
@samcv updated

File: racket.html.markdown (hunk 2 above)
Comment (samcv) [inline racket.html.markdown]: Okay, you should probably mention that the ` backtick mark is acts as a quasiquote in Racket then.

Comment (samcv) [inline racket.html.markdown]: I'm not saying to refer to not refer to it as a quasiquote. Say something like:
Quasiquotes with commas can be used to evaluate functions, using the backtick character

Or something to that effect.

Comment (Derrreks) [inline racket.html.markdown]: That makes sense. I updated it

File: racket.html.markdown [hunk 3]
```diff
@@ -164,10 +164,17 @@ my-pet ; => #<dog>
 (cons 1 (cons 2 (cons 3 null))) ; => '(1 2 3)
 ;; `list' is a convenience variadic constructor for lists
 (list 1 2 3) ; => '(1 2 3)
-;; and a quote can also be used for a literal list value
+;; a quote can also be used for a literal list value
 '(1 2 3) ; => '(1 2 3)
+;; a quasiquote (represented by the backtick character) with commas 
+;; can be used to evaluate functions
```
Comment (vendethiel) [inline racket.html.markdown]: It's a bit simplistic. It can be used to allow splices in general

**Comment by samcv** [comment]
@Derrreks Thanks! Merged.

**Comment by Derrreks** [comment]
Great, thank you!

**Pull request merged**